Integrations/CentralBankIntegrationLib/RussianCBInfo.cs

```diff
@@ -33,7 +34,7 @@ public List<ExchangeRate> GetCurrencies()
             var rates = new List<ExchangeRate>();
             for (int i = 0; i < data.Currencies.Count; i++)
             {
-                rates.Add(new ExchangeRate { Code = data[i].CharCode, Value = data[i].ValueInDigits });
+                rates.Add(new ExchangeRate { Code = data[i].CharCode, Value = decimal.Parse(data[i].ValueInString.Replace(',', '.')) });
```

Comment:
I have an exception here. Why do you need to change this line
System.FormatException: 'Input string was not in a correct format.'

Reply:
> I have an exception here. Why do you need to change this line
> System.FormatException: 'Input string was not in a correct format.'

There were some culture conflicts because my OS is in English I guess, I'll fix it so it works everywhere then

Reply:
> I have an exception here. Why do you need to change this line
> System.FormatException: 'Input string was not in a correct format.'

Fixed (i guess)